tests/connect_redirect/connect_redirect_tests.cpp

```diff
@@ -64,6 +131,20 @@ _get_protocol_from_string(std::string protocol)
     REQUIRE(false);
 }
 
+static execution_mode_t
+_get_execution_mode(std::string& execution_mode_string)
+{
+    if (execution_mode_string == "" || execution_mode_string == "Admin") {
```

Comment:
Why "Admin" instead of "Administrator"?  We try to avoid abbreviations where possible.

Reply:
Fixed.